### PR TITLE
Initial tutorials and audit trail in configuration documents.

### PR DESCRIPTION
**Pull request by mandy-chessell**

### CODE REVIEW

File: open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/task-building-egeria-source.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+```bash
+
+$ mvn clean install
+
+``` 
+The build can take 5-15 minutes depending on the speed and load on your machine.
+However eventually you will see the message:
+
+```bash
```
Comment (cmgrote) [inline open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/task-building-egeria-source.md]:
```suggestion
```text
```

File: open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/task-installing-egeria.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+# Installing ODPi Egeria
+
+The [egeria build process](task-building-egeria-source.md) creates the
+distribution files for Egeria in the open-metadata-distribution module.
+The assemblies are located from the egeria build directory as follows:
+
+```text
```
Comment (cmgrote) [inline open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/task-installing-egeria.md]:
```suggestion
```bash
```

File: open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/task-installing-egeria.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+The name of the files is determined by the release level of the code that you
+[downloaded from GitHub](task-downloading-egeria-source.md).  In this example,
+the release is `egeria-0.3-SNAPSHOT`.
+
+Create a directory for the OMAG Server and copy the tar file into it.
+The two commands shown below create a directory at the same level in the
+file system as the egeria build library and copies the omag server into it.
+The then changes to the new directory and lists its contents.
+
+```text
```
Comment (cmgrote) [inline open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/task-installing-egeria.md]:
```suggestion
```bash
```

File: open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/task-installing-egeria.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+$ mkdir ../../../../egeria-install
+$ cp egeria*-distribution.tar.gz ../../../../egeria-install
+$ cd ../../../../egeria-install
+$ ls
+egeria-0.3-SNAPSHOT-distribution.tar.gz
+```
+
+It is now possible to unpack the tar file.
+
+```text
```
Comment (cmgrote) [inline open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/task-installing-egeria.md]:
```suggestion
```bash
```

File: open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/task-installing-egeria.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+```
+
+Under `server` is a directory for the
+[OMAG Server](../../../open-metadata-publication/website/omag-server).
+This is the Egeria standalone server.
+
+Change to the omag server's directory.
+
+```text
```
Comment (cmgrote) [inline open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/task-installing-egeria.md]:
```suggestion
```bash
```

File: open-metadata-resources/open-metadata-tutorials/omag-server-tutorial/task-starting-the-omag-server.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+# Starting the OMAG Server
+
+The [OMAG Server's installation directory](../building-egeria-tutorial/task-installing-egeria.md)
+contains a Java Archive (Jar) file for the server and a directory of resources.
+
+```text
```
Comment (cmgrote) [inline open-metadata-resources/open-metadata-tutorials/omag-server-tutorial/task-starting-the-omag-server.md]:
```suggestion
```bash
```

File: open-metadata-resources/open-metadata-tutorials/omag-server-tutorial/task-starting-the-omag-server.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+The OMAG server is started with the `java` command.
+Ensure you have a Java runtime at Version 8 (Update 151) or above installed on your machine.
+Check the version of Java you have with the command `java -version`
+(You can download Java from [AdoptOpenJDK](https://adoptopenjdk.net/) and select `OpenJDK 8 (LTS)`
+and the `HotSpot` version for your operating system.
+You only need the JRE but select the JDK if you expect to also write some Java code.)
+
+Start the OMAG Server as follows 
+
+```text
```
Comment (cmgrote) [inline open-metadata-resources/open-metadata-tutorials/omag-server-tutorial/task-starting-the-omag-server.md]:
```suggestion
```bash
```

**Comment by cmgrote** [comment]
Loving the start to the tutorials! 👍  Just a few inline comments for consistency on syntax highlighting for the shell command sections...

File: open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/README.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+and the `HotSpot` version for your operating system.  You need the JDK rather than the JRE.)
+* Ensure you have [Apache Maven](https://maven.apache.org/) 3.5.0 or above installed.
+This is used to orchestrate the ODPi Egeria build.
+* Ensure you have `git` installed.  On MacOS, this is part of the `Xcode` suite.
+For other operating systems there are [various choices](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+
+
+## Tutorial tasks
+
+* [Downloading the Egeria source from GitHub](task-downloading-egeria-source.md)
```
Comment (cmgrote) [inline open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/README.md]:
```suggestion
1. [Downloading the Egeria source from GitHub](task-downloading-egeria-source.md)
```

File: open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/README.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+* [Building the Egeria source with Apache Maven](task-building-egeria-source.md)
```
Comment (cmgrote) [inline open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/README.md]:
```suggestion
1. [Building the Egeria source with Apache Maven](task-building-egeria-source.md)
```

File: open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/README.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+* [Installing Egeria ready for use](task-installing-egeria.md)
```
Comment (cmgrote) [inline open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/README.md]:
```suggestion
1. [Installing Egeria ready for use](task-installing-egeria.md)
```

File: open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/task-installing-egeria.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+# Installing ODPi Egeria
+
+The [egeria build process](task-building-egeria-source.md) creates the
+distribution files for Egeria in the open-metadata-distribution module.
+The assemblies are located from the egeria build directory as follows:
```
Comment (cmgrote) [inline open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/task-installing-egeria.md]:
```suggestion
The assemblies are located from the `egeria` build directory as follows:
```

File: open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/task-installing-egeria.md [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+
+```
+The name of the files is determined by the release level of the code that you
+[downloaded from GitHub](task-downloading-egeria-source.md).  In this example,
+the release is `egeria-0.3-SNAPSHOT`.
+
+Create a directory for the OMAG Server and copy the tar file into it.
+The two commands shown below create a directory at the same level in the
+file system as the egeria build library and copies the omag server into it.
+The then changes to the new directory and lists its contents.
```
Comment (cmgrote) [inline open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/task-installing-egeria.md]:
```suggestion
It then changes to the new directory and lists its contents.
```

File: open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/README.md (hunk 10 above)
Comment (mandy-chessell) [reply open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/README.md]: Not sure I understand this edit

File: open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/README.md (hunk 8 above)
Comment (mandy-chessell) [reply open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/README.md]: Don't understand this change - seems to be switching the last bullet in list ot a number

Comment (cmgrote) [reply open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/README.md]: Meant to change the bullet list into a numbered sequence, as the ordering seemed important — not sure what happened to the other edits

File: open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/README.md (hunk 10 above)
Comment (cmgrote) [reply open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/README.md]: As with other: suggesting change from bullet list to numbered sequence

**Pull request merged**